sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentAnalysisClient/DocumentAnalysisClientLiveTests.cs

```diff
@@ -1242,18 +1241,16 @@ public async Task StartAnalyzeDocumentPopulatesExtractedReceiptJpg(bool useStrea
 
             Assert.NotNull(document.Fields);
 
-            Assert.True(document.Fields.ContainsKey("ReceiptType"));
```

Comment:
Not supported anymore. The receipt type will be returned in the `AnalyzedDocument.DocType`.